desktop/public_desktop/preload_preferences.js

```diff
@@ -0,0 +1,15 @@
+const { ipcRenderer, contextBridge } = require( 'electron' );
+
+const sendChannels = [ 'preferences-changed', 'preferences-changed-' ];
```

Comment:
.include won't pick up `preferences-changed-BLAHBLAH`

Reply:
Good catch - fixed in 8506b829dbdefd82ce1b9853a23dd9573a6b896e.